snuba/clickhouse/formatter/query.py

```diff
@@ -73,6 +80,13 @@ def _visit_simple_query(
         self, data_source: ProcessableQuery[Table]
     ) -> FormattedSubQuery:
         assert isinstance(data_source, Query)
+        if data_source.is_delete():
```

Comment:
I actually don't know that this is possible, we wouldn't have `DELETE FROM` as a subquery